Documentation/RC1SupportedScenarios.md

```diff
@@ -0,0 +1,24 @@
+WCF in RC1
+===========================
+
+WCF Features
+------------
+
+
+![Image](http://findicons.com/files/icons/573/must_have/16/check.png) -- Supported: Release quality
+
+![Image](http://findicons.com/files/icons/1155/diagona/16/104.png) --  Supported: Not fully tested
+
+![Image](http://findicons.com/files/icons/573/must_have/16/add.png) -- Not Yet Supported: Future release
+
+![Image](http://findicons.com/files/icons/573/must_have/16/delete.png) -- Not Supported: Will not support
+
+
+
+|Scenario |UWP   |Windows |Linux |Mac OS X  |
+|---------|------|--------|------|----------|
```

Comment:
Apparently emoji's do work here on GitHub (https://github.com/markdown-it/markdown-it-emoji) unfortunately I can't use them in Visual Studio Code, there may be a plug-in I don't know about, there may also be some editor GitHub/markdown-it recommends to assist during development.